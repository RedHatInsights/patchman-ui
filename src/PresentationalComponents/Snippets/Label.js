import propTypes from 'prop-types';
import React from 'react';
import './Label.scss';

const Label = ({ children }) => <span className='patchman-label'>{children}</span>;

Label.propTypes = {
  children: propTypes.any,
};

export default Label;
