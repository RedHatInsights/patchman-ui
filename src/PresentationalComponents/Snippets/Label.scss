span {
    &.patchman-label {
        display: inline-block;
        font-size: var(--pf-v5-global--FontSize--sm);
        font-weight: var(--pf-v5-global--FontWeight--bold);
        line-height: var(--pf-v5-global--LineHeight--sm);
        color: var(--pf-v5-global--Color--100);
    }
}
