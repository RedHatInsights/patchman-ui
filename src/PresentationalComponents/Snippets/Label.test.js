import Label from './Label';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Label.js', () => {
    it('Should match the snapshot', () => {
        const wrapper = shallow(<Label message = {<div></div>} />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
