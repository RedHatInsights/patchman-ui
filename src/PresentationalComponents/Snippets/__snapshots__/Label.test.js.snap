// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label.js Should match the snapshot 1`] = `
<span
  className="patchman-label"
/>
`;
